sharedPrompts/VALIDATION_IMPROVEMENTS.md

```
@@ -0,0 +1,169 @@
+- **구현체**: `BlogCommandValidator`, `DocumentCommandValidator`, `EmailCommandValidator`, `TextCommandValidator`, `ImageCommandValidator`
+
+## 개선 제안
+
+### 1. 명명 규칙 개선
+
+#### 문제점
+- `BlogValidator`와 `BlogCommandValidator`가 혼동을 야기
+- 두 시스템의 목적이 명확하지 않음
+
+#### 제안
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)